client/src/components/Attachment/AttachmentCard.js

```diff
@@ -0,0 +1,128 @@
+import { gql } from "@apollo/client"
+import { Icon } from "@blueprintjs/core"
+import { IconNames } from "@blueprintjs/icons"
+import API from "api"
+import ConfirmDestructive from "components/ConfirmDestructive"
+import LinkTo from "components/LinkTo"
+import PropTypes from "prop-types"
+import React from "react"
+import { Card } from "react-bootstrap"
+import { toast } from "react-toastify"
+import pdf from "resources/newPDF.svg"
+import "./Attachment.css"
+
+const GQL_DELETE_ATTACHMENT = gql`
+  mutation ($uuid: String!) {
+    deleteAttachment(uuid: $uuid)
+  }
+`
+
+const AttachmentCard = ({
+  file,
+  show,
+  remove,
+  setError,
+  setRemove,
+  uploadedList,
+  setUploadedList
+}) => {
+  const returnFileSize = number => {
+    if (number < 1024) {
+      return `${number} bytes`
+    } else if (number >= 1024 && number < 1048576) {
+      return `${(number / 1024).toFixed(1)} KB`
+    } else if (number >= 1048576) {
+      return `${(number / 1048576).toFixed(1)} MB`
+    }
+  }
+
+  return (
+    <div key={file.uuid} style={{ width: "20%" }}>
+      <Card>
+        <div
+          className="imagePreview info-show card-image"
+          style={{
+            backgroundSize: file.mimeType.includes("pdf") ? "50px" : "cover",
+            backgroundImage: file.mimeType.includes("pdf")
+              ? `url(${pdf})`
+              : file.content.includes("data")
+                ? `url(${file.content})`
+                : `url(data:${file.mimeType};base64,${file.content})`
+          }}
+        >
+          <div className="file-info image-info">
+            <div style={{ display: "grid" }}>
+              <LinkTo
+                className="detail-btn"
+                modelType="Attachment"
+                model={file}
+              >
+                {" "}
+              </LinkTo>
+            </div>
+          </div>
+        </div>
+        <Card.Body className={`p-1 ${!show ? "d-none" : "d-block"}`}>
+          <Card.Title style={{ fontSize: "15px" }} className="info-line">
+            {file?.fileName?.substring(0, 8)}...
+            <span>{returnFileSize(file?.content?.length)}</span>
```

Comment:
We should probably introduce another accessor `Attachment.getContentLength()` which we fill with `length(content)` in the database query (so you can get the length without having to get the entire file contents just to show that number).